_posts/2021-03-14-curating/curating.Rmd

```diff
@@ -0,0 +1,329 @@
+
+1. Practice with TweetDeck in advance. Literally, force yourself to tweet from Tweetdeck at least a week before your curation. TweetDeck is very different than the standard Twitter interface, and it took me a few days to get used to it.
+
+2. Figure out how I wanted to style shared code in advance - my first couple of days would have gone smoother with this. 
+
+3. Prepare a tweet in honor of any holidays or significant events coinciding with your curation week. One regret that I do have from my curating week is failing to explicitly knowledge Black History Month as I was tweeting in February. I wish had prepared at least one tweet or better amplified the voices of black members of the R community while I had the large platform. 
+
+# Fleeting fame
+
+When curating, your tweets in the moment are highly visible. But what persists afterward is fairly anonymous as your tweets are not linked to your personal profile unless you tag yourself. In a weird way, it actually becomes a safe place to put yourself out there with 
+questions you might not have been comfortable asking from your own personal account. Take advantage of this fleeting fame not just to share your knowledge but also to ask your questions. 
+
+# Support your curators
+
+Just because a twitter account has >20K followers, the likes, re-tweets, and comments don't come automatically. You still have to earn engagement with your content. Many of the tweets I sent had little engagement, and that is okay. Supporting your curators by engaging with their tweets or sending notes of encouragement is _much_ appreciated. I thank **everyone** who engaged with me during my curation, with a special shout out to Alison Hill who re-energized me mid-week with comments on the R-Ladies bloggers [thread](https://twitter.com/WeAreRLadies/status/1362021673239785473){target="_blank"}. 
+
+I cannot emphasize this enough: every like, re-tweet, comment, and direct message helps. In addition, if you have curated in the past, consider sending new curators a personal welcome message and an invitation to ask you any questions.
+
+
+# Reflection
+
+Between prepared and ad-hoc content and discussions with followers, I tweeted a lot! (At least for me.)  Here is a [summary thread](https://twitter.com/spcanelon/status/1363518469782843396){target="_blank"} of my tweets for the week. My tweets were not perfect, and that is okay. I messed up threads, had typos, and shared deprecated code, among other things. Check out my [blooper reel](https://twitter.com/PipingHotData/status/1364183660744896513){target="_blank"} for tweets that I bungled.
+
+Serving as a curator was intimidating and time consuming, but I am very glad I did it. Many good things have happened as direct result of that week, including: 
+
+* [discussing](https://twitter.com/WeAreRLadies/status/1363144545677017089){target="_blank"} comparisons between `{blogdown}` and `{distill}` with Alison Hill 
```

Comment:
I read that it's best to add periods after each item in lists. I can't find the source for my claim though.